### PR TITLE
Closes #1486 [NTR] Fallopian tube ciliated cell

### LINKED ISSUE #1486: [NTR] Fallopian tube ciliated cell
**Preferred term label:**
Fallopian tube ciliated cell

**Synonyms**
oviduct ciliated cell
uterine tube ciliated epithelial cell

**Definition**
An epithelial ciliated cell that is part of the fallopian tube epithelium. Fallopian tube ciliated cells are found mainly on the apex of the mucosal folds and contribute together with the peristaltic contraction to the self propulsion of spermatozoa, the transport of ovum during ovulation and the transport of the fertilized ovum to the intramural fallopian tube. Ciliated cells have a columnar shape and contain an oval or round nucleus, often located perpendicular or parallel to the long axis of the cell [PMID: 16565155]. 

**Parent cell type term:**
multi-ciliated epithelial cell (CL:0005012)

**What is the anatomical structure that the cell is a part of?**
ciliated columnar oviduct epithelium (UBERON:0007589)

**Your nano-attribution (ORCID)**
0000-0002-0819-0473

**Any additional notes or concerns**

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Closes #1486 [NTR] Fallopian tube ciliated cell

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
Hi @paolaroncaglia, would you mind taking a look at this, especially the synonyms since you have been in previous discussions involving 'fallopian tube' vs 'oviduct'? https://sourceforge.net/p/geneontology/ontology-requests/8397/

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -29729,6 +29730,18 @@ AnnotationAssertion(rdfs:label obo:CL_4030006 "fallopian tube secretory epitheli
...
 
+# Class: obo:CL_4030007 (fallopian tube ciliated cell)
+
+AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/contributor> "http://orcid.org/0000-0002-0819-0473"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:16565155"^^xsd:string) obo:IAO_0000115 obo:CL_4030007 "An epithelial ciliated cell that is part of the fallopian tube. This cell type is found mainly on the apex of the mucosal folds and contributes, together with peristaltic contractions, to the self-propulsion of spermatozoa, the transport of ovum during ovulation and the transport of the fertilized ovum to the intramural fallopian tube. A fallopian tube ciliated cell has a columnar shape and contains an oval or round nucleus, often located perpendicular or parallel to the long axis of the cell."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030007 <http://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_4030007 "2022-03-08T12:40:19Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030007 "oviduct ciliated cell"^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:29738]: @bvarner-ebi I'd make this a broad synonym please, as not all oviducts have Fallopian tubes (e.g. in non-mammals)

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -29729,6 +29730,18 @@ AnnotationAssertion(rdfs:label obo:CL_4030006 "fallopian tube secretory epitheli
...
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4030007 "uterine tube ciliated epithelial cell"^^xsd:string)
```
Comment (paolaroncaglia) [inline src/ontology/cl-edit.owl:29739]: @bvarner-ebi this one's fine

**Review by paolaroncaglia — COMMENTED** [review]
@bvarner-ebi I added a comment.
P.S. A SourceForge ticket from 2011 - _now_ I feel old! :-D

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:29738]: Thank you! In that case, I also update the synonym type for the 'fallopian tube secretory epithelial cell' as well.

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi I added a comment.
> P.S. A SourceForge ticket from 2011 - _now_ I feel old! :-D

I see it as an accumulation of knowledge and credibility :)

**Comment by bvarner-ebi** [comment]
@paolaroncaglia, could you confirm your review so that I can proceed with the merge? My permissions require reviews after every commit before I can merge.

**Comment by paolaroncaglia** [comment]
@bvarner-ebi 

> @paolaroncaglia, could you confirm your review so that I can proceed with the merge? My permissions require a review after every commit before I can merge.

Sorry, I missed the second review notification. Thanks for the reminder!

**Pull request merged**